ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1969,6 +1972,8 @@ def get_ret_and_flattened_np_array(fn, *args, test_compile: bool = False, **kwar
     def map_fn(x):
         if _is_frontend_array(x):
             return x.ivy_array
+        if isinstance(x, ivy.functional.frontends.numpy.ndarray):
```

Comment:
I think this is old changes from old commit, please also remove this (repetition with the existing code)

Reply:
which line ? 4th line?